docs/PUBLISHING.md

```diff
@@ -10,6 +10,6 @@ Marked uses [semantic-release](https://github.com/semantic-release/semantic-rele
 
 We follow [semantic versioning](https://semver.org) where the following sequence is true `[major].[minor].[patch]`:
 
-1. **Major:** There is at least one change to the public API or a break from the [CommonMark](https://spec.commonmark.org/current/) or [GFM](https://github.github.com/gfm/) spec.
+1. **Major:** There is at least one change to the public API or a break from the [CommonMark](https://spec.commonmark.org/current/) or [GFM](https://github.github.com/gfm/) spec. Only [current and LTS](https://nodejs.org/en/about/releases/) Node.js versions are supported at any point in time.  A drop in support for a Node.js version may not result in a semver major bump to Marked.
```

Comment:
I don't think we plan to do this.

We should do semver major when dropping a version of Node.js and bump the engines field

https://github.com/markedjs/marked/blob/e781b18b0faf8d23691c88769638027a33fcaaec/package.json#L87

Reply:
Would be great if that happened but I wasn't able to convince everyone on the team :)

Unless something changed since then?

Reply:
@styfle Currently we test Node v12, Node lts (v14), and Node latest (v16). When Node LTS changes to v16 we will be testing Node v12, Node lts (v16), and Node latest (v17), so Node v14 will no longer be tested. If we had a team of people working on this every day I think we could task someone with updating that, but realistically there are three-ish people working on marked, whenever we have extra time.

I think it is good to set the expectations low. Ideally we won't break Node versions that aren't tested and we we will only update the engines field on semver major releases but I still like saying, realistically, we could break older node versions in patches since only lts and latest are guaranteed to be tested and the best fix might be for you to update Node.